### ISSUE TITLE
Instructions for manually updating the patch file

### ISSUE THREAD

**Issue by dhirschfeld** [conda-forge/fastapi-feedstock#13]
```python
from github3 import GitHub

version = '0.42.0'

gh = GitHub()
repo = gh.repository('tiangolo', 'fastapi')
contents = repo.file_contents('/pyproject.toml', ref=version)
lines = contents.decoded.splitlines()

header = [
    b"--- a/pyproject.toml\r\n",
    b"+++ /dev/null\r\n",
    b"@@ -1,%d +0,0 @@\r\n" % len(lines),
]
lines = [b'-%s\r\n' % line for line in lines]
lines = header + lines
with open('./recipe/no-pyproject.toml.patch', 'wb') as f:
    f.writelines(lines)
```

**Comment by igortg** [comment]
Why this patch is required?

**Comment by dhirschfeld** [comment]
No longer required since it was fixed in #25.
